README.md.tpl

```diff
@@ -33,6 +33,8 @@ Configure your {brandname} cluster by specifying values in the `deploy.*` sectio
 | `deploy.expose.host` | Specifies the hostname where the {ingress} is exposed, if required. | `""` | |
 | `deploy.expose.annotations` | Adds annotations to the service that exposes {brandname} on the network. | `{}` | - |
 | `deploy.logging.categories` | Configures {brandname} cluster log categories and levels. | `{}` | - |
+| `deploy.podLabels` | Adds labels to every pod created. | `{}` | - |
+| `deploy.svcLabels` | Adds labels to every service created. | `{}` | - |
```

Comment:
We should update the `deploy.resourceLabels` description to make it explicit that `resourceLabels` are applied to all created resources. Something like: "Adds labels to all {brandname} resources including pods and services."

Reply:
Done!